site/content/en/docs/Guides/metrics.md

```diff
@@ -173,24 +178,3 @@ kubectl port-forward deployments/grafana 3000 -n metrics
 Open a web browser to [http://127.0.0.1:3000](http://127.0.0.1:3000), you should see Agones [dashboards](#grafana-dashboards) after login as admin.
 
 > Makefile targets `make grafana-portforward`,`make kind-grafana-portforward` and `make minikube-grafana-portforward`.
-
-## Adding more metrics
```

Comment:
Curious - reason for removing this? (Not saying we shouldn't, just wondering what the motivation was)

Reply:
I had a talk last week with @jkowalski and we both think opencensus is already an abstraction there is no issue in using metrics directly where it matter, for example in the gameserver package we could have a metric.go file.

Originally this documentation comes from our discussion and we thought it would be good to avoid spreading metric code in our code base, but ultimately I think this makes thing more difficult to write.

Reply:
Okily. :+1: